<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomSettings</class>
 <widget class="QDialog" name="CustomSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="custom_ok">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="custom_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="Advanced">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>WARNING</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="2" column="0">
        <widget class="QLabel" name="Warning_text">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Don't edit these settings unless you know what you are doing. Editing these settings may break the program or make it unusable.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Warning_label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>WARNING:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="VideoTab">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0">
        <widget class="QLabel" name="ConversionLabel">
         <property name="text">
          <string>Threads for conversion</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="ResolutionLabel">
         <property name="text">
          <string>Camera Resolution</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QCheckBox" name="live555Copy">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="FlipLabel">
         <property name="text">
          <string>Flip views</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="OpenGLLabel">
         <property name="text">
          <string>Use OpenGL for views</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="CopyingLabel">
         <property name="text">
          <string>Copy in Live555</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="format_box">
         <property name="maximumSize">
          <size>
           <width>164</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="Input">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="flip">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="yuv_threads">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="Processing">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Processing</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="opengl">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="resolution">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>164</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="format_label">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="framerate_box"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="framerate_label">
         <property name="text">
          <string>Framerate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="KvazaarTab">
      <attribute name="title">
       <string>HEVC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="2">
        <widget class="QSpinBox" name="qp">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quantization parameter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>51</number>
         </property>
         <property name="value">
          <number>27</number>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Threads</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="VPS">
         <property name="text">
          <string>VPS period</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="Intra">
         <property name="text">
          <string>Intra period</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QLabel" name="OpenHEVC">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>OpenHEVC Parameters</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="3">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="kvz_threads_label">
         <property name="text">
          <string>Threads</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QTableWidget" name="custom_parameters">
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <column>
          <property name="text">
           <string>Parameter name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="wpp">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wavefront Parallel Processing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="vps">
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="WPP">
         <property name="text">
          <string>WPP</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1" colspan="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="intra">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>64</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="kvz_threads">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="maxLength">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="QLineEdit" name="openhevc_threads">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="QP">
         <property name="text">
          <string>QP</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QCheckBox" name="slices">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="slice_label">
         <property name="text">
          <string>Slices (depends on WPP)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="Preset">
         <property name="text">
          <string>preset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="Kvazaar">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Kvazaar Parameters</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="preset">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>ultrafast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>superfast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>veryfast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>faster</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>slow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>slower</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>veryslow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>placebo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Parameter name</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="parameter_name"/>
       </item>
       <item row="11" column="2">
        <widget class="QLineEdit" name="parameter_value"/>
       </item>
       <item row="11" column="0">
        <widget class="QPushButton" name="add_parameter">
         <property name="text">
          <string>Add parameter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AudioTab">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sample Rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>WARNING: Audio settings functionality not implemented</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="AudioLine">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="AudioLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Audio</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="channels">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="channels_label">
         <property name="text">
          <string>Channels</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Codec</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>Opus</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Raw pcmu</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>preset</tabstop>
  <tabstop>kvz_threads</tabstop>
  <tabstop>qp</tabstop>
  <tabstop>wpp</tabstop>
  <tabstop>vps</tabstop>
  <tabstop>intra</tabstop>
  <tabstop>slices</tabstop>
  <tabstop>custom_parameters</tabstop>
  <tabstop>parameter_name</tabstop>
  <tabstop>openhevc_threads</tabstop>
  <tabstop>custom_ok</tabstop>
  <tabstop>custom_close</tabstop>
  <tabstop>opengl</tabstop>
  <tabstop>resolution</tabstop>
  <tabstop>framerate_box</tabstop>
  <tabstop>live555Copy</tabstop>
  <tabstop>format_box</tabstop>
  <tabstop>Advanced</tabstop>
  <tabstop>flip</tabstop>
  <tabstop>yuv_threads</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>channels</tabstop>
  <tabstop>comboBox_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
